add-to-hosts.sh

```diff
@@ -3,7 +3,7 @@
 # function that takes kubernetes ingress name and domain name and ands it to /etc/hosts
 
 function add_ingress_to_hosts() {
-    echo -e  "$(kubectl get ingress $1 -o jsonpath='{.status.loadBalancer.ingress[0].ip}') $2" | sudo tee -a /etc/hosts
+    echo -e  "$(kubectl get ingress $1 -n$2 -o jsonpath='{.status.loadBalancer.ingress[0].ip}') $3" | sudo tee -a /etc/hosts
```

Comment:
this extends the add_ingress_to_hosts() function with an additional input $2 - the namespace